menu.kando.Kando.yml

```diff
@@ -0,0 +1,35 @@
+id: menu.kando.Kando
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+command: AppRun
+finish-args:
+  - --share=ipc
+  - --device=dri
+  - --socket=x11
+  - --socket=wayland
+  - --talk-name=org.freedesktop.Flatpak
```

Comment:
needs an exception https://docs.flathub.org/docs/for-app-authors/linter#exceptions 

https://docs.flathub.org/docs/for-app-authors/linter#finish-args-flatpak-spawn-access

Reply:
Shall I create a corresponding PR at this point? Or is this only required at a later stage?

Reply:
I created a corresponding pull request: https://github.com/flathub-infra/flatpak-builder-lint/pull/544. Let me know if something is missing there :smiley: